### ISSUE TITLE
Elite controller rapid fire

### ISSUE THREAD

**Issue by brillante1** [the-red-team/XboxOneRapidFire#2]
@justcallmekoko 

![Screenshot_20191022-231747__01__01](https://user-images.githubusercontent.com/56652679/70340433-13b0e880-1851-11ea-827d-04f32be0ca8a.jpg)
![Screenshot_20191206-172729__01](https://user-images.githubusercontent.com/56652679/70340438-157aac00-1851-11ea-849e-663da0f99250.jpg)
![Screenshot_20191206-172611__01](https://user-images.githubusercontent.com/56652679/70340443-190e3300-1851-11ea-95cb-fb6f75546c7e.jpg)

**Comment by justcallmekoko** [comment]
These are great pictures. Where is the yellow wire leading?
Where did you learn these solder points? Were you just poking around or did you find some documentation online?

**Comment by brillante1** [comment]
I Found on internet 😁, the yellow is for te led light on the controller.. what i would to ask you, indon't know anything about making a code.. is there a possibility to make a code where you van put multiple modes in? ( Mode 1:fast rapid fire mode2: slower rapid fire etc...?

**Comment by brillante1** [comment]
![Screenshot_20191124-211614__01](https://user-images.githubusercontent.com/56652679/70345845-3c3edf80-185d-11ea-8810-477b7b25da14.jpg)

**Comment by justcallmekoko** [comment]
@brillante1 Excellent! That is the same point I am using for the LED in the elite controller now with the new mod.
So with the new mod, I have taken mode changes and fire rate changes into consideration, but there is a twist to it that makes it much better than current mods. I also have akimbo working on it.
So keep an eye out for the new mod and it will have exactly what you are asking for and more.

**Comment by brillante1** [comment]
I can't wait for new mod!! 😁👌🏼👌🏼

**Comment by brillante1** [comment]
Do you know if there is a possibility to put off the led's from the Arduino?

**Comment by brillante1** [comment]
![Screenshot_20191211-175908__01](https://user-images.githubusercontent.com/56652679/70642776-1ee48980-1c40-11ea-8b15-9ee0a689f604.jpg)
![Screenshot_20191211-175859__01](https://user-images.githubusercontent.com/56652679/70642783-22781080-1c40-11ea-8af4-2fd8c8e1e00a.jpg)

**Comment by brillante1** [comment]
Without delay 👌🏼

**Comment by justcallmekoko** [comment]
@brillante1 What are the screenshots for? 

When you say "put off the leds from the arduino" are you referring to the LEDs on the arduino itself of the LED on the controller?

**Comment by brillante1** [comment]
On the Arduino...

**Comment by justcallmekoko** [comment]
@brillante1 You can probably just desolder the LEDs on the arduino.

In that documentation you found with the solder points on the elite controller, did they include the location of the left trigger solder point?

Just a heads up. I posted the video about the new mod [here](https://www.youtube.com/watch?v=Ci7KqMRJ9D4) and the github repo is [here](https://github.com/justcallmekoko/ROTA)

**Comment by brillante1** [comment]
![Screenshot_20191212-152324__02](https://user-images.githubusercontent.com/56652679/70720081-77c12a00-1cf3-11ea-8964-bfc2c7f43d52.jpg)


Left Trigger 😉👍🏼

**Comment by justcallmekoko** [comment]
Excellent. I am probably going to go buy a damn elite controller today just to try this out. Thanks for sharing these pictures!

**Comment by brillante1** [comment]
Yes i know it's me with the awnser about the expensive taxes to europe 😅

**Comment by brillante1** [comment]
No problem! Your welcome!

**Comment by justcallmekoko** [comment]
@brillante1 Oh hahaha yea man sorry again about the those crazy prices. How much would the mod have to cost for it to be worth it? I still can't believe those taxes and shipping costs.

**Comment by brillante1** [comment]
It's hard to find space in the elite controller.. the only empty space i Found was between LT trigger and the vibration.. 😉

**Comment by justcallmekoko** [comment]
I'm curious if I could manage to sandwich it between the boards

**Comment by brillante1** [comment]
The price of the mod is a fair price! 
The customs take the price of the item + delivery costs.. and that price they take +/- 24% of it

**Comment by brillante1** [comment]
I used a shrink sleeve around the mod beceause i had a electrical breakdown ...

**Comment by justcallmekoko** [comment]
Well damn. How good are you at programming and soldering the ESP8266? You can do what I did when I first started making this mod and...
1. Flash the code and SPIFFS image onto an ESP8266 with a frogo board or something
2. solder surface mount resistors to the necessary pins to make it boot properly
3. solder it to the controller

I actually have really shitty pictures of this exact thing on my instagram in the really early photos. It is crude but it works.

I also used heat shrink around the pro micro which I forgot to mention in the video

**Comment by brillante1** [comment]
![Screenshot_20191212-154108__01](https://user-images.githubusercontent.com/56652679/70721468-e69f8280-1cf5-11ea-8de5-59f7c8e7082c.jpg)
The Mark Points you better don't touch

**Comment by brillante1** [comment]
Haha, my cousin did the code.. he is very good in programming  😅

**Comment by justcallmekoko** [comment]
lol well get him to do the code again. If you want to try the bare bones rota mod like I said, let me know so I can post the SPIFFS files

**Comment by brillante1** [comment]
I Will ask to do the code 😉 i Will search for the ESP8266

**Comment by brillante1** [comment]
![Screenshot_20191212-161628__01](https://user-images.githubusercontent.com/56652679/70724429-dfc73e80-1cfa-11ea-9654-13244399fdfc.jpg)
Is this the right one?

**Comment by justcallmekoko** [comment]
No search for the ESP8266 ESP-12E

**Comment by justcallmekoko** [comment]
![image](https://user-images.githubusercontent.com/25190487/70727985-9f51cb80-1cce-11ea-879f-061425087368.png)

**Comment by brillante1** [comment]
Oke, Found it..! And how it works to put the code on it? Just connect it with wifi?

**Comment by justcallmekoko** [comment]
I like to use [this](https://www.tindie.com/products/fred_iot/esp8266-frogboard-frogo-pins-esp-12e-dev-board/)

**Comment by brillante1** [comment]
Thanks! Never seen this tool before 😁
